### PR TITLE
Pin to pytorch#117182 (tags/v2.2.0-rc8)

### PR DESCRIPTION
**Pull request by yeounoh**
Pin to https://github.com/pytorch/pytorch/pull/117182 --> tags/v2.2.0-rc8

### CODE REVIEW

File: torch_patches/.torch_pin [hunk 1]
```diff
@@ -0,0 +1 @@
+#117182
```
Comment (JackCaoG) [inline torch_patches/.torch_pin:1]: please follow the example of https://github.com/pytorch/xla/blob/r2.1/.torch_pin. I think we should just pin it to releases/2.2 branch for now and update it to a tag after release it out.

**Comment by zpcore** [comment]
I think it should be `tags/v2.2.0-rc8`.

**Review by ManfeiBai — APPROVED** [review]
vote to `tags/v2.2.0-rc8`

File: torch_patches/.torch_pin (hunk 1 above)
Comment (ManfeiBai) [reply torch_patches/.torch_pin:1]: do we wanna update to `tags/v2.2.0-rc8` directly?

Comment (yeounoh) [reply torch_patches/.torch_pin]: Thank you, changed!

File: torch_patches/.torch_pin [hunk 2]
```diff
@@ -0,0 +1 @@
+tags/v2.2.0-rc8
```
Comment (JackCaoG) [inline torch_patches/.torch_pin:1]: let's move this outside of `torch_patches`. I think that's what we want to do for release branches.

Comment (yeounoh) [reply torch_patches/.torch_pin]: Done!

**Review by JackCaoG — APPROVED** [review]

**Review by ManfeiBai — APPROVED** [review]
LGTM, Thanks, Yeounoh!

**Review by zpcore — APPROVED** [review]

**Pull request merged**